06-files-input-and-output/remove_files.py

```diff
@@ -2,33 +2,43 @@
 
 import os
 
-def find_files_in_folder(path, extention, file_max_size):
+def find_files_in_folder(path, extension, file_max_size):
+    print(
+        "Searching for files with .{} extension in \"{}\" and maximum size of {}bytes".format(
+            extension, path, file_max_size
```

Comment:
Each time you break up a call to a function to several lines you need to put each argument passed to the function in a new line, like what you wrote above:
```
the_best_function_ever(
    extension="jpg", 
    file_max_size=200,
    path="/",
)
```
If you pass to the function variables with the exact same names as the function's arguments, you can skip the `=` signs:
```
the_best_function_ever(
    extension, 
    file_max_size,
    path,
)
```